pkg/dockerutil/dockerutils.go

```diff
@@ -842,6 +856,26 @@ func CheckDockerVersion(versionConstraint string) error {
 	return nil
 }
 
+// HasCompatibleDockerAPIVersion checks if the Docker API version of the host system is compatible with the required minimum version.
+func HasCompatibleDockerAPIVersion(minimumAPIVersion string) (bool, error) {
+	const (
+		versionMatrixURL   = "https://docs.docker.com/reference/api/engine/#api-version-matrix"
+		troubleshootingURL = "https://ddev.readthedocs.io/en/stable/users/install/docker-installation/#testing-and-troubleshooting-your-docker-installation"
+		helpfulLinks       = "\nSee API version matrix at %s\nTroubleshoot at %s"
```

Comment:
After checking several failures I'm not entirely sure that this link is helpful, but it's fine. This should be an unusual situation.

Reply:
Do you mean `versionMatrixURL` or `troubleshootingURL` ?

Reply:
The helpfulLinks with the API version matrix. Basically in our situation people don't need a matrix, they just need to update. And.. as discussed elsewhere, it's going to catch the Docker server version anyway.